static/src/javascripts/projects/common/modules/ui/relativedates.js

```diff
@@ -0,0 +1,233 @@
+        hours = Math.round(delta / 3600);
+        return hours + getSuffix('h', format, hours);
+    } else if (extendedFormatting && isWithinPastWeek(then)) {
+        days = Math.round(delta / 3600 / 24);
+        return days + getSuffix('d', format, days);
+    } else if (isYesterday(then)) {
+        // yesterday
+        return `Yesterday${withTime(then)}`;
+    } else if (delta < 5 * 24 * 60 * 60) {
+        // less than 5 days
+        return (
+            [
+                dayOfWeek(then.getDay()),
+                then.getDate(),
+                monthAbbr(then.getMonth()),
+                then.getFullYear(),
+            ].join(' ') + (opts.showTime ? withTime(then) : '')
+        );
+    }
+    return (
+        [then.getDate(), monthAbbr(then.getMonth()), then.getFullYear()].join(
+            ' '
+        ) + (opts.showTime ? withTime(then) : '')
+    );
+};
+
+const findValidTimestamps = (): bonzo =>
+    // `.blocktime time` used in blog html
+    $('.js-timestamp, .js-item__timestamp');
+
+const replaceLocaleTimestamps = (html: ?Element): void => {
+    const cls = 'js-locale-timestamp';
+    const context = html || document;
+
+    $(`.${cls}`, context).each(el => {
+        let datetime;
+        const $el = bonzo(el);
+        const timestamp = parseInt($el.attr('data-timestamp'), 10);
+
+        if (timestamp) {
+            datetime = new Date(timestamp);
+            el.innerHTML = `${pad(datetime.getHours())}:${pad(
+                datetime.getMinutes()
+            )}`;
+            $el.removeClass(cls);
+        }
+    });
+};
+
+const replaceValidTimestamps = (opts: RelativeDateOptions = {}): void => {
+    findValidTimestamps().each(el => {
+        let targetEl;
+        const $el = bonzo(el);
+
+        const // Epoch dates are more reliable, fallback to datetime for liveblog blocks
+        timestamp =
+            parseInt($el.attr('data-timestamp'), 10) || $el.attr('datetime');
+
+        const datetime = new Date(timestamp);
+
+        const relativeDate = makeRelativeDate(datetime.getTime(), {
+            // NOTE: if this is in a block (blog), assume we want added time on > 1 day old dates
+            showTime: bonzo($el.parent()).hasClass('block-time'),
+            format: $el.attr('data-relativeformat'),
+            notAfter: opts.notAfter,
+        });
+
+        if (relativeDate) {
+            // If we find .timestamp__text (facia), use that instead
+            targetEl = $el[0].querySelector('.timestamp__text') || $el[0];
+            if (!targetEl.getAttribute('title')) {
+                targetEl.setAttribute('title', bonzo(targetEl).text());
+            }
+            targetEl.innerHTML = relativeDate;
+        } else if (opts.notAfter) {
+            $el.addClass('modern-hidden');
+        }
+    });
+};
+
+// DEPRECATED: Bindings
+['related', 'autoupdate'].forEach(module => {
+    mediator.on(`modules:${module}:render`, replaceValidTimestamps);
+});
```

Comment:
Nothing in the codebase is listening for these events, these bindings can be safely deleted.